### DISCUSSION TITLE
HMS-1600 mit 2 Modulen und 1ner Batterie "gemischt" betreiben?

### DISCUSSION THREAD

**Discussion by Stefanodavinci** [hoylabs/OpenDTU-OnBattery#1454] [General]
Hallo!

Meine bisherige Anlage bestand aus einem HMS-800 an welchen eine 24v 100ah Batterie angeschlossen war. Diese wurde von 2 Modulen in Südausrichtung und einen Victron 100/30 geladen. Nun habe ich mir einen HMS-1600 sowie 2 weiter Module zugelegt. Meine Idee ist, je ein Modul in Ost- bzw. Westausrichtung an den WR anzuschließen und an die beiden übrigen Mppt meine Bestandsanlage zu migrieren. 

Was mein ihr, funktioniert das so? 
Welche Einstellungen muss ich im DPL Setup wählen? (WR wir von PV /Batterie betrieben)

Besten Dank schonmal.

**Comment by Sorbat** [comment]
Wenn ich deine Beschreibung richtig verstehe, dann ist es ähnlich wie hier: 

https://github.com/hoylabs/OpenDTU-OnBattery/discussions/1302#discussioncomment-10801595

**Comment by Stefanodavinci** [comment]
Ja, ist ähnlich außer das es sich bei mir um 4 Eingänge bzw. Mppts handelt. Der erste Punkt im FAQ macht es ja schon ziemlich deutlich daß es wohl nicht funktionieren wird. Ich dachte das es zusammen mit HM Firmware Update, was erlaubt die Ausgangsleistung zu begrenzen, vllt. funktioniert. Außerdem bin ich davon ausgegangen das nicht alle 4 Mppt zusammen berücksichtigt werden.

**Comment by schlimmchen** [comment]
Man kann die MPPTs nicht einzeln steuern. Du müsstest mindestens dafür sorgen, dass die Batterie sich abschaltet, und zwar nicht erst aus der Not heraus, sondern zu einen sinnvollen Zeitpunkt. Denn der DPL wird entweder den ganzen Inverter stoppen, wenn die Batterie leer ist, obwohl an zwei Eingängen Leistung zu holen wäre, oder er wird ihn laufen lassen, bis die tiefentladen ist und das BMS eingreift, was es zu vermeiden gilt.

Geschickt wäre es, einen Inverter mit zwei Eingängen an der Batterie zu haben und den anderen an den Modulen. Oder auch sehr gut: alle Module am Solarladeregler und den Inverter an der Batterie.

**Reply by Stefanodavinci** [reply]
Vielen Dank für die Antwort. Ich hab auch nochmal darüber nachgedacht wie bzw. warum mein Aufbau nicht funktionieren sollte und mit Hilfe von #1302 (comment) bin ich beim selben Szenario gelandet welches du beschreibst. Rein technisch würde es natürlich funktionieren aber das Problem ist wie gesagt der DPL. Was die Lösungen angeht, 2 separate WR wären sicher die eleganteste Losung, Frage dazu: kann man die WR priorisieren oder verteilt der DPL den Bedarf zu gleichen Teilen? Mir ist noch eingefallen dass man das Problem mit der Tiefenentladung recht einfach mit einem Victron smart battery protect lösen kann oder liege ich da falsch?

**Reply by schlimmchen** [reply]
Der DPL versucht (natürlich) möglichst die nötige Leistung bei solarbetriebenen WR zu "bestellen". Nur was dann noch darüber hinaus gebraucht wird, wird bei batteriebetriebenen WR als Limit eingestellt.

> Mir ist noch eingefallen dass man das Problem mit der Tiefenentladung recht einfach mit einem Victron smart battery protect lösen kann oder liege ich da falsch?

Der schaltet dann ab an einer konservativ hohen Spannung? Wenn der dann abschaltet, haben zwei Eingänge des WR keine Leistung. Das muss dann der WR kompensieren. Die neue Firmware kann das scheinbar. Andernfalls wird der WR stets zu wenig Leistung liefern, weil der DPL erwartet, dass das Limit, das der DPL einstellt, auch als Ausgangsleistung erreicht wird bei batteriebetriebenen WR. Also nochmal: Das wird nicht unterstützt, es ist in der Logik des DPL nirgends berücksichtigt.
